src/tests/ftest/util/pool_create_all_base.py

```diff
@@ -44,10 +44,13 @@ def create_pool(self, index):
 
         return end_time - start_time
 
-    def create_one_pool(self):
+    # pylint: disable=dangerous-default-value
+    def create_one_pool(self, ranks=[]):
```

Comment:
We should not ignore this. You can just do this to default to `[]`
```
    def create_one_pool(self, ranks=None):
        if ranks is None:
            ranks = []
```

Reply:
Good catch and I really prefer your proposal :)

- [ ] remove pylint disbale pragma